docs/pagination.md

```diff
@@ -104,6 +104,26 @@ export default DRFSerializer.extend({
 });
 ```
 
+## Cursor Pagination
+
+To use `CursorPagination`, override `extractPageNumber` in the serializer to extract the `cursor`.
+
+```js
+// app/serializer/drf.js
+
+import DRFSerializer from 'ember-django-adapter/serializers/drf';
+
+export default DRFSerializer.extend({
+  extractPageNumber: function(url) {
```

Comment:
Not a blocker here, but I think we (all of us) should probably start using ES6 notation in our docs.